src/Microsoft.AspNet.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -374,11 +375,16 @@ internal static PropertyValidationInfo GetPropertyValidationInfo(ModelBindingCon
             return validationInfo;
         }
 
-        internal void ProcessDto(ModelBindingContext bindingContext, ComplexModelDto dto)
+        internal ModelValidationNode ProcessDto(ModelBindingContext bindingContext, ComplexModelDto dto)
```

Comment:
It's a little strange to have `ProcessDTO` return the validation node. Maybe just return the MBR here?